auth/oras-go/client.go

```diff
@@ -0,0 +1,42 @@
+package orasgo
```

Comment:
Instead of `oras-go` and `package orasgo`, we can just have `oras` folder and `package oras`.

Reply:
changed to `oras` folder and `package oras`